### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by godswift** [jmalarcon/Win11ClockToggler#14]
just installed this, when i try to hide the date and time it hides the network connection and sound buttons as well
![image](https://user-images.githubusercontent.com/34440773/189424203-64486605-559b-458b-bd2a-a70e000a165d.png)
![image](https://user-images.githubusercontent.com/34440773/189424262-59b1853a-73ee-417f-b9aa-d5d15ac56555.png)

I am not sure if it in conflict of translucentTB and ElevenClock

**Comment by jmalarcon** [comment]
If your carefully read the readme file on the cover, what Win11ClockToggler does is hiding not only the clock but everything that is in that area. That includes the system icons:

![image](https://user-images.githubusercontent.com/1775322/189491466-b9375571-8999-48d7-bace-0f8ebb3eca30.png)

What happens in your case is that ElevenClock hides the original Windows Clock and puts its own. When my app runs it hides the UI element that contains the clock, but it's not aware of the one added by ElevenClock. So the net result is that only the system icons are hidden.

Summing up: **it won't work with Elevenclock, just with the native Windows clock**.

I didn't know of the existence of ElevenClock. I'll check its code to see if it gives me a clue on how to hide just the native clock.

**Comment by jmalarcon** [comment]
I've check the code of ElevenClock and what they do is to position a new window above the native clock, hiding it below. It doesn't find the real position of the clock: just asume a width and height and put its own clock above it. In fact, in my System, it hides part of the system icons. So it doesn't hide or change the native clock: just creates a new one and tries to hide the original positioning itself where it is supposed to be.

So, as I correctly supposed, if you're using that app, Win11ClockToggler correctly hides the native clock (and system icons), but the extra ElevenClock window remains in place.

Just don't use ElevenClock if you want to hide the native clock.
